CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing Guidelines
+You have to follow two rules only when contributing:
+
+* Produce readable code
+* **No Tests -> No Merge!**
+
+# Development Environment Setup
+If you plan to contribute a feature or bugfix to this project or just the want to have a closer look at the sources,
+the following steps describe the setup of the development environment.
+ 
+1. Start an Eclipse IDE v4.4 (Luna) or later that has the [Plug-in Development Environment (PDE)](https://www.eclipse.org/pde/) installed.
+ If you want to use a fresh Eclipse installation, you can download the _Eclipse IDE for Eclipse Committers_ package from the [Eclipse Downloads](https://www.eclipse.org/downloads/packages/) page.
+It contains everything you need.
+ 
+2. Clone the git repository from https://github.com/MoreUnit/MoreUnit-Eclipse.git
+ 
+3. The necessary projects are located at the root of the repository. 
+Import all of them into your workspace.
+
+4. The `org.moreunit.build` project contains suitable [Target Platform Definitions](http://help.eclipse.org/juno/index.jsp?topic=%2Forg.eclipse.pde.doc.user%2Fconcepts%2Ftarget.htm).
+Open one of them and select the _Set as Target Platform_ link.
+
+
+Now you are ready to hack the sources.
+If you whish to verify the setup, you can run the test suite. The launch configuration to run all tests is called _All Tests_

```

Comment:
Sorry, this sentence is plain wrong - I've taken it from another project of mine.
As far as I can see, MoreUnit currently has no launch configuration to run all tests (include the SWTBot tests). I would rewrite the phrase as:
> If you wish to verify the setup, you can run the Maven/Tycho build with the _mvn clean install_ launch configuration.